identity-service/src/routes/cognito.js

```diff
@@ -56,15 +56,47 @@ module.exports = function (app) {
     const { id: sessionId, customer_reference: handle, status } = data
 
     try {
+      // check that this identity has not already been used by another account before proceeding to save the score
+      let shouldFail = false
```

Comment:
could be better to have a more descriptive var

let cognitoIdentityAlreadyExists = false